docs/pull_request_template.md

```diff
@@ -7,7 +7,10 @@
 
 - [ ] Unit tests
 - [ ] Automated tests (e.g. Preflight)
+- [ ] Confirm this change does not bypass the SSO security model
```

Comment:
```suggestion
- [ ] Confirm that this change does not bypass the security model or best practices
```
We're going to need, separately, to document these and can include "SSO" in that list.